### PR TITLE
Task solution

### PR DESCRIPTION
**Pull request by ayasinska**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,23 @@
...
+  const PRICE_PER_DAY = 40;
+  const LONG_RENT = 7;
+  const LONG_RENT_DISCOUNT = 50;
+  const SHORT_RENT = 3;
+  const SHORT_RENT_DISCOUNT = 20;
+
+  let basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_RENT) {
+    return basePrice - LONG_RENT_DISCOUNT;
```
Comment (vlasiuksasha) [inline src/calculateRentalCost.js:40]: The LONG_RENT_DISCOUNT should be subtracted from the basePrice only when the long rent duration is exactly equal to 7. If the rent duration is more than 7, the discount should be more.

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,23 @@
...
+  }
+
+  if (days >= SHORT_RENT) {
+    return basePrice - SHORT_RENT_DISCOUNT;
```
Comment (vlasiuksasha) [inline src/calculateRentalCost.js:44]: The SHORT_RENT_DISCOUNT should be subtracted from the basePrice only when the short rent duration is exactly equal to 3. If the rent duration is more than 3 but less than 7, the discount should be more.

**Review by vlasiuksasha — CHANGES_REQUESTED** [review]
check the comments

File: src/calculateRentalCost.js (hunk 1 above)
Comment (ayasinska) [reply src/calculateRentalCost.js:40]: https://mate.academy/chats/27190

File: src/calculateRentalCost.js (hunk 2 above)
Comment (ayasinska) [reply src/calculateRentalCost.js:44]: https://mate.academy/chats/27190

**Review by 4ksandrey — CHANGES_REQUESTED** [review]
try to resend task

**Pull request closed without merging**